src/Uno.Toolkit.UI/Controls/ZoomContentControl/ZoomContentControl.Properties.cs

```diff
@@ -0,0 +1,477 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Numerics;
+using Uno.Disposables;
+using Uno.Extensions.Specialized;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using System.Threading.Tasks;
+
+#if IS_WINUI
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Controls.Primitives;
+using Microsoft.UI.Xaml.Data;
+using Microsoft.UI.Xaml.Input;
+using Microsoft.UI.Xaml.Media;
+using Microsoft.UI.Xaml.Navigation;
+using Microsoft.UI.Input;
+#else
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+using Windows.UI.Input;
+using Windows.Devices.Input;
+#endif
+
+namespace Uno.Toolkit.UI;
+
+public partial class ZoomContentControl
+{
+	#region DependencyProperty: HorizontalOffset
+
+	/// Identifies the HorizontalOffset dependency property.
+	public static DependencyProperty HorizontalOffsetProperty { get; } = DependencyProperty.Register(
+		nameof(HorizontalOffset),
+		typeof(double),
+		typeof(ZoomContentControl),
+		new PropertyMetadata(default(double), OnHorizontalOffsetChanged));
+
+	/// Gets or sets the horizontal offset for panning the content.
+	public double HorizontalOffset
+	{
+		get => (double)GetValue(HorizontalOffsetProperty);
+		set => SetValue(HorizontalOffsetProperty, value);
+	}
+
+	#endregion
+
+	#region DependencyProperty: HorizontalScrollValue
```

Comment:
As we removed VerticalScrollValue, shouldn't we remove this one as well?

Reply:
`HorizontalScrollValue` & `HorizontalOffset` are not always in sync like the `VerticalScrollValue` & `VerticalOffset`

HorizontalScrollValue is -1 * HorizontalOffset

I am not sure if we should remove the HorizontalScrollValue?

Reply:
My feeling is that there should only be `HorizontalOffset`, but check with @Xiaoy312

Reply:
the relationship is messy...
```
private async void UpdateHorizontalScrollBarValue()
    HorizontalScrollValue = -1 * HorizontalOffset;
    await RaiseRenderedContentUpdated();

private void ScrollH_Scroll(object sender, ScrollEventArgs e)
    HorizontalOffset = -1 * e.NewValue;

<Grid>
    <Canvas x:Name="PART_Canvas">
        <ContentPresenter x:Name="PART_Presenter">
            <ContentPresenter.RenderTransform>
                <TransformGroup>
                    <TranslateTransform X="{TemplateBinding HorizontalOffset}"
                                        Y="{TemplateBinding VerticalOffset}" />
    <ScrollBar x:Name="PART_ScrollV" Value="{TemplateBinding VerticalOffset}" />
    <ScrollBar x:Name="PART_ScrollH" Value="{TemplateBinding HorizontalScrollValue}" />
```
we are probably best to keep only one set of H/V-something-value/offset properties. probably best to keep the ScrollBar values are Horizontal/VerticalScrollValue, and drop the -Offset counterpart.
and for the TranslateTransform, just use x:Name to grab the instance and updating the value directly with the -1 multiplier consideration to align scroll bar direction with the translation

Reply:
naming the remaining set as `-Offset` or `-ScrollValue` is just preference, or what you think makes more sense